ui/src/app/pages/homepage/data-access-requirements.spec.tsx

```diff
@@ -113,14 +113,11 @@ describe('DataAccessRequirements', () => {
         expect(activeModule).toEqual(enabledModules[1]);
 
         // update this if the order changes
-        expect(activeModule).toEqual(AccessModule.ERACOMMONS)
+        expect(activeModule).toEqual(AccessModule.RASLINKLOGINGOV)
     });
 
-    // TODO(RW-7301)
-    // restore the intended module order [2FA, RAS, ERA] from the temporary version [2FA, ERA, RAS]
-
     it('should return the second enabled module (RAS, not ERA) from getActiveModule' +
```

Comment:
change to (ERA, not RAS) and make RAS the disabled module.  Update other comments here from ERA to RAS